apps/services/auth/ids-api/infra/identity-server.ts

```diff
@@ -196,5 +196,6 @@ export const serviceSetup = (services: {
       'nginx-ingress-external',
       'user-notification',
       'portals-admin',
+      'datadog',
```

Comment:
Remember to put this on all the services in this namespace

Reply:
done, good point